VTCareService/src/main/java/edu/vt/cs/vtcare/meetingservices/models/ZoomMeetingResponse.java

```diff
@@ -1,24 +1,31 @@
 package edu.vt.cs.vtcare.meetingservices.models;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
I'm sure there's a reason for this, I'm curious not questioning it was needed, but why.

Reply:
When creating an appointment via ZOOM API, they return a bunch of things that are not needed. We only need a few properties, hence when converting that response to our local object we need to ignore those extra properties.